src/components/search/Search.jsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const Search = ({ search, onSubmit, onChange }) => (
+  <>
+    <label htmlFor="search">Search</label>
+    <input
+      id="search"
+      type="search"
+      name="search"
+      value={search}
+      onChange={onChange}
+    />
+    <button onClick={onSubmit} data-testid="button">Search</button>
+  </>
```

Comment:
You should put this all in a form instead of a fragment and use onSubmit rather than an onClick for the button